src/state/chains/chain.state.ts

```diff
@@ -226,12 +226,24 @@ export const chainClient$ = state(
   ),
 )
 export const client$ = state(chainClient$.pipe(map(({ client }) => client)))
-
 export const canProduceBlocks$ = state(
   client$.pipe(
     switchMap((client) => client._request("rpc_methods", [])),
     map((response) => response.methods.includes("dev_newBlock")),
+    liftSuspense(),
     catchError(() => [false]),
+    sinkSuspense(),
```

Comment:
We had a bug here, when switching between fork with chopsticks and not, this would cause to return "false"